adafruit_pcf8563/pcf8563.py

```diff
@@ -75,26 +79,41 @@ class PCF8563:
     alarm = i2c_bcd_alarm.BCDAlarmTimeRegister(
         0x09, has_seconds=False, weekday_shared=False, weekday_start=0
     )
-    """Alarm time for the alarm."""
+    """Alarm time for the alarm. Note that the value of the seconds-fields
+    is ignored, i.e. alarms only fire at full minutes. For short-term
+    alarms, use a timer instead."""
 
     alarm_interrupt = i2c_bit.RWBit(0x01, 1)
     """True if the interrupt pin will output when alarm is alarming."""
 
     alarm_status = i2c_bit.RWBit(0x01, 3)
     """True if alarm is alarming. Set to False to reset."""
 
+    clockout_enabled = i2c_bit.RWBit(0x0D, 7)
+    """True if clockout is enabled (default). To disable clockout, set to False"""
+
+    clockout_frequency = i2c_bits.RWBits(2, 0x0D, 0)
+    """Clock output frequencies generated. Default is 32.768kHz.
+    Possible values are as shown (selection value - frequency).
+    00 - 32.768khz
+    01 - 1.024kHz
+    10 - 0.032kHz (32Hz)
+    11 - 0.001kHz (1Hz)
+    """
+
+    CLOCKOUT_FREQ_32KHZ = const(0b00)
+    """Clock frequency of 32 KHz"""
+    CLOCKOUT_FREQ_1KHZ = const(0b01)
+    """Clock frequency of  4 KHz"""
+    CLOCKOUT_FREQ_32HZ = const(0b10)
+    """Clock frequency of 32 Hz"""
+    CLOCKOUT_FREQ_1HZ = const(0b11)
+    """Clock frequency of 1 Hz"""
+
```

Comment:
Want to remove this since there is a separate class for?

Reply:
good point, I will update the PR